packages/sw-broadcast-cache-update/src/lib/broadcast-cache-update.js

```diff
@@ -27,47 +27,34 @@ import {defaultHeadersToCheck, defaultSource} from './constants';
  * For efficiency's sake, the underlying response bodies are not compared;
  * only specific response headers are checked.
  *
- * @example <caption>Added as a "plugin" to a `RequestWrapper` to
- * automatically dispatch messages on a cache update</caption>
+ * This class is meant to be agnostic about the service worker environment it's
```

Comment:
Nit: Could you rephrase this to focus on "This class can be used to <insert use here> and can be used in your service worker without any of the other sw-helper modules."